src/org/opendatakit/briefcase/ui/automation/AutomationPanel.java

```diff
@@ -68,6 +68,7 @@ private void generateScript(String scriptName, AutomationConfiguration configura
     String briefcaseJar = isWindows() ? "%JAR%" : "$JAR";
     List<String> pullInstructions = forms.getSelectedForms()
         .stream()
+        .filter(FormStatus::isPullSelected)
```

Comment:
This three lines could be `forms.getPullForms()` or `forms.getSelectedForPulling()`

Reply:
Same for the other blocks